roles/addons/nic_nmcli/readme.rst

```diff
@@ -10,6 +10,11 @@ The role also cover routes definitions on interfaces.
 This role provides all features availables in the main nmcli module.
 Please refer to `nmcli module documentation <https://docs.ansible.com/ansible/latest/collections/community/general/nmcli_module.html>`_ .
 
+.. warning:
+  This role needs **latest** (2.0.0) nmcli.py module, from
+  https://github.com/ansible-collections/community.general/
+  ansible-collections:main branch.
+
```

Comment:
The community.general 2.0.0 should be [available end of January 2021](https://mailchi.mp/redhat/the-bullhorn-15). In the meantime, I would not recommend to install a development version of this collection unless one really needs to use the new features introduced in this PR (routes and zones).
I would reflect this in the warning message: _If you plan to configure routes or zones with this role, you will have to install the development version of the community.general collection which is available at https://github.com/ansible-collections/community.general/ until the General Availability of the 2.0.0 release._

If bluebanquise 1.4 should land prior to this collection, I would indicate both features are tech-preview.

Reply:
Updated the readme